### ISSUE TITLE
🔧 react/bridging/CallbackWrapper.h' file not found with v3🔧 

### ISSUE THREAD

**Issue by Filocava99** [cuvent/react-native-vision-camera#2203]
### How were you trying to build the app?

I was trying to build my Expo app after setting use_Framework to static but now I get the following error: `🔧 react/bridging/CallbackWrapper.h' file not found`. I created a new issue because all the existing ones are specifically for version 2 of RNVC.

### Full build logs

```tsx
Showing Recent Errors Only

Build target VisionCamera of project Pods with configuration Debug

CompileC /Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Intermediates.noindex/Pods.build/Debug-iphoneos/VisionCamera.build/Objects-normal/arm64/JSINSObjectConversion.o /Users/filippocavallari/Desktop/Replate/ReplateApp/node_modules/react-native-vision-camera/ios/Frame\ Processor/JSINSObjectConversion.mm normal arm64 objective-c++ com.apple.compilers.llvm.clang.1_0.compiler (in target 'VisionCamera' from project 'Pods')
    cd /Users/filippocavallari/Desktop/Replate/ReplateApp/ios/Pods
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -x objective-c++ -ivfsstatcache /Users/filippocavallari/Library/Developer/Xcode/DerivedData/SDKStatCaches.noindex/iphoneos17.0-21A325-1e1d10ddc0bdcd738a98873f48b793a5.sdkstatcache -target arm64-apple-ios12.4 -fmessage-length\=0 -fdiagnostics-show-note-include-stack -fmacro-backtrace-limit\=0 -fno-color-diagnostics -std\=c++17 -stdlib\=libc++ -fobjc-arc -fmodules -gmodules -fmodules-cache-path\=/Users/filippocavallari/Library/Developer/Xcode/DerivedData/ModuleCache.noindex -fmodules-prune-interval\=86400 -fmodules-prune-after\=345600 -fbuild-session-file\=/Users/filippocavallari/Library/Developer/Xcode/DerivedData/ModuleCache.noindex/Session.modulevalidation -fmodules-validate-once-per-build-session -Wnon-modular-include-in-framework-module -Werror\=non-modular-include-in-framework-module -fmodule-name\=VisionCamera -Wno-trigraphs -fpascal-strings -O0 -fno-common -Wno-missing-field-initializers -Wno-missing-prototypes -Werror\=return-type -Wdocumentation -Wunreachable-code -Wno-implicit-atomic-properties -Werror\=deprecated-objc-isa-usage -Wno-objc-interface-ivars -Werror\=objc-root-class -Wno-arc-repeated-use-of-weak -Wimplicit-retain-self -Wno-non-virtual-dtor -Wno-overloaded-virtual -Wno-exit-time-destructors -Wduplicate-method-match -Wno-missing-braces -Wparentheses -Wswitch -Wunused-function -Wno-unused-label -Wno-unused-parameter -Wunused-variable -Wunused-value -Wempty-body -Wuninitialized -Wconditional-uninitialized -Wno-unknown-pragmas -Wno-shadow -Wno-four-char-constants -Wno-conversion -Wconstant-conversion -Wint-conversion -Wbool-conversion -Wenum-conversion -Wno-float-conversion -Wnon-literal-null-conversion -Wobjc-literal-conversion -Wshorten-64-to-32 -Wno-newline-eof -Wno-selector -Wno-strict-selector-match -Wundeclared-selector -Wdeprecated-implementations -Wno-c++11-extensions -Wno-implicit-fallthrough -DPOD_CONFIGURATION_DEBUG\=1 -DDEBUG\=1 -DCOCOAPODS\=1 -DPOD_CONFIGURATION_DEBUG\=1 -DDEBUG\=1 -DSK_METAL\=1 -DSK_GANESH\=1 -DVISION_CAMERA_ENABLE_FRAME_PROCESSORS\=true -D_LIBCPP_ENABLE_CXX17_REMOVED_UNARY_BINARY_FUNCTION -DOBJC_OLD_DISPATCH_PROTOTYPES\=0 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS17.0.sdk -fstrict-aliasing -Wprotocol -Wdeprecated-declarations -Winvalid-offsetof -g -fvisibility-inlines-hidden -Wno-sign-conversion -Winfinite-recursion -Wmove -Wcomma -Wblock-capture-autoreleasing -Wstrict-prototypes -Wrange-loop-analysis -Wno-semicolon-before-method-body -Wunguarded-availability -index-store-path /Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Index.noindex/DataStore -iquote /Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Intermediates.noindex/Pods.build/Debug-iphoneos/VisionCamera.build/VisionCamera-generated-files.hmap -I/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Intermediates.noindex/Pods.build/Debug-iphoneos/VisionCamera.build/VisionCamera-own-target-headers.hmap -I/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Intermediates.noindex/Pods.build/Debug-iphoneos/VisionCamera.build/VisionCamera-all-non-framework-target-headers.hmap -ivfsoverlay /Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Intermediates.noindex/Pods.build/Debug-iphoneos/VisionCamera.build/all-product-headers.yaml -iquote /Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Intermediates.noindex/Pods.build/Debug-iphoneos/VisionCamera.build/VisionCamera-project-headers.hmap -I/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/VisionCamera/include -I/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/ReactCommon-Samples/ReactCommon_Samples.framework/Headers -I/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/ReactCommon/ReactCommon.framework/Headers/react/nativemodule/core -I/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/ReactCommon-Samples/ReactCommon_Samples.framework/Headers/platform/ios -I/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/React-NativeModulesApple/React_NativeModulesApple.framework/Headers -I/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/React-graphics/React_graphics.framework/Headers/react/renderer/graphics/platform/ios -I/Users/filippocavallari/Desktop/Replate/ReplateApp/ios/Pods/Headers/Public -I/Users/filippocavallari/Desktop/Replate/ReplateApp/ios/Pods/Headers/Public/FBLazyVector -I/Users/filippocavallari/Desktop/Replate/ReplateApp/ios/Pods/Headers/Public/RCTRequired -I/Users/filippocavallari/Desktop/Replate/ReplateApp/ios/Pods/Headers/Public/React-callinvoker -I/Users/filippocavallari/Desktop/Replate/ReplateApp/ios/Pods/Headers/Public/React-runtimeexecutor -I/Users/filippocavallari/Desktop/Replate/ReplateApp/ios/Pods/Headers/Public/hermes-engine -I../../node_modules/react-native-vision-camera/cpp -I/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Intermediates.noindex/Pods.build/Debug-iphoneos/VisionCamera.build/DerivedSources-normal/arm64 -I/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Intermediates.noindex/Pods.build/Debug-iphoneos/VisionCamera.build/DerivedSources/arm64 -I/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Intermediates.noindex/Pods.build/Debug-iphoneos/VisionCamera.build/DerivedSources -F/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/VisionCamera -F/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/DoubleConversion -F/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/RCT-Folly -F/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/RCTTypeSafety -F/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/React-Codegen -F/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/React-Core -F/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/React-NativeModulesApple -F/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/React-RCTAnimation -F/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/React-RCTBlob -F/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/React-RCTImage -F/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/React-RCTLinking -F/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/React-RCTNetwork -F/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/React-RCTSettings -F/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/React-RCTText -F/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/React-RCTVibration -F/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/React-cxxreact -F/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/React-debug -F/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/React-hermes -F/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/React-jsi -F/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/React-jsiexecutor -F/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/React-jsinspector -F/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/React-logger -F/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/React-perflogger -F/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/React-utils -F/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/ReactCommon -F/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/SocketRocket -F/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/Yoga -F/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/fmt -F/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/glog -F/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/libevent -F/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/react-native-worklets-core -F/Users/filippocavallari/Desktop/Replate/ReplateApp/ios/Pods/hermes-engine/destroot/Library/Frameworks/universal -F/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/XCFrameworkIntermediates/hermes-engine/Pre-built -include /Users/filippocavallari/Desktop/Replate/ReplateApp/ios/Pods/Target\ Support\ Files/VisionCamera/VisionCamera-prefix.pch -MMD -MT dependencies -MF /Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Intermediates.noindex/Pods.build/Debug-iphoneos/VisionCamera.build/Objects-normal/arm64/JSINSObjectConversion.d --serialize-diagnostics /Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Intermediates.noindex/Pods.build/Debug-iphoneos/VisionCamera.build/Objects-normal/arm64/JSINSObjectConversion.dia -c /Users/filippocavallari/Desktop/Replate/ReplateApp/node_modules/react-native-vision-camera/ios/Frame\ Processor/JSINSObjectConversion.mm -o /Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Intermediates.noindex/Pods.build/Debug-iphoneos/VisionCamera.build/Objects-normal/arm64/JSINSObjectConversion.o -index-unit-output-path /Pods.build/Debug-iphoneos/VisionCamera.build/Objects-normal/arm64/JSINSObjectConversion.o

In file included from /Users/filippocavallari/Desktop/Replate/ReplateApp/node_modules/react-native-vision-camera/ios/Frame Processor/JSINSObjectConversion.mm:25:
In file included from /Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/ReactCommon/ReactCommon.framework/Headers/react/nativemodule/core/ReactCommon/TurboModuleUtils.h:16:
/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/ReactCommon/ReactCommon.framework/Headers/react/nativemodule/core/ReactCommon/CallbackWrapper.h:11:10: fatal error: 'react/bridging/CallbackWrapper.h' file not found
#include <react/bridging/CallbackWrapper.h>
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/ReactCommon/ReactCommon.framework/Headers/react/nativemodule/core/ReactCommon/CallbackWrapper.h:11:10: note: did not find header 'bridging/CallbackWrapper.h' in framework 'react' (loaded from '/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/React-Core')
1 error generated.

/Users/filippocavallari/Library/Developer/Xcode/DerivedData/replateapp-djjburazmxijpmazqstsfnxwqwcq/Build/Products/Debug-iphoneos/ReactCommon/ReactCommon.framework/Headers/react/nativemodule/core/ReactCommon/CallbackWrapper.h:11:10: 'react/bridging/CallbackWrapper.h' file not found
```
```


### Project dependencies

```json
"dependencies": {
    "@apollo/client": "^3.8.4",
    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.18.6",
    "@babel/plugin-proposal-optional-chaining": "^7.21.0",
    "@babel/plugin-transform-arrow-functions": "^7.22.5",
    "@babel/plugin-transform-shorthand-properties": "^7.22.5",
    "@babel/plugin-transform-template-literals": "^7.22.5",
    "@babel/preset-env": "^7.1.6",
    "@react-buddy/ide-toolbox": "^2.4.0",
    "@react-native-async-storage/async-storage": "^1.19.3",
    "@react-native-firebase/app": "^18.6.1",
    "@react-native-masked-view/masked-view": "^0.2.9",
    "@react-navigation/native": "^6.1.7",
    "@react-navigation/stack": "^6.3.17",
    "@reduxjs/toolkit": "^1.9.7",
    "@types/react": "~18.2.14",
    "@viro-community/react-viro": "npm:react-viro-sw",
    "apollo3-cache-persist": "^0.14.1",
    "axios": "^1.5.0",
    "expo": "~49.0.11",
    "expo-asset": "^8.10.1",
    "expo-asset-utils": "^3.0.0",
    "expo-av": "^13.6.0",
    "expo-barcode-scanner": "^12.7.0",
    "expo-blur": "^12.6.0",
    "expo-build-properties": "~0.8.3",
    "expo-device": "^5.6.0",
    "expo-file-system": "^15.6.0",
    "expo-font": "^11.6.0",
    "expo-haptics": "^12.6.0",
    "expo-linear-gradient": "^12.5.0",
    "expo-location": "^16.3.0",
    "expo-media-library": "^15.6.0",
    "expo-modules-core": "^1.5.11",
    "expo-navigation-bar": "^2.5.0",
    "expo-notifications": "^0.23.0",
    "expo-sensors": "^12.5.0",
    "expo-splash-screen": "^0.22.0",
    "expo-status-bar": "~1.6.0",
    "expo-updates": "~0.18.13",
    "graphql": "^16.8.1",
    "i18next": "^23.5.1",
    "lottie-react-native": "^6.4.0",
    "react": "18.2.0",
    "react-dom": "^18.2.0",
    "react-i18next": "^13.2.2",
    "react-native": "0.72.4",
    "react-native-circular-progress-indicator": "^4.4.2",
    "react-native-color-matrix-image-filters": "^6.0.3",
    "react-native-config": "^1.5.1",
    "react-native-device-info": "^10.11.0",
    "react-native-draggable-flatlist": "^4.0.1",
    "react-native-gesture-handler": "^2.13.1",
    "react-native-get-random-values": "^1.9.0",
    "react-native-gradients": "^2.0.1",
    "react-native-linear-gradient": "^2.8.3",
    "react-native-localize": "^3.0.2",
    "react-native-modal": "^13.0.1",
    "react-native-reanimated": "^3.5.4",
    "react-native-safe-area-context": "^4.7.2",
    "react-native-screens": "^3.25.0",
    "react-native-skeleton-placeholder": "^5.2.4",
    "react-native-svg": "^13.14.0",
    "react-native-svg-transformer": "^1.1.0",
    "react-native-swipe-gestures": "^1.0.5",
    "react-native-swiper": "^1.6.0",
    "react-native-vector-icons": "^10.0.0",
    "react-native-view-shot": "^3.7.0",
    "react-native-vision-camera": "^3.6.8",
    "react-native-webview": "^13.6.0",
    "react-native-worklets-core": "^0.2.2",
    "react-redux": "^8.1.3",
    "typescript": "^5.1.3"
  },
```
```


### VisionCamera Version

3.6.8

### Target platforms

iOS

### Operating system

MacOS

### Can you build the VisionCamera Example app?

No, I cannot build the Example app either

### Additional information

- [X] I am using Expo
- [X] I have enabled Frame Processors (react-native-worklets-core)
- [X] I have read the [Troubleshooting Guide](https://react-native-vision-camera.com/docs/guides/troubleshooting)
- [X] I agree to follow this project's [Code of Conduct](https://github.com/mrousavy/react-native-vision-camera/blob/main/CODE_OF_CONDUCT.md)
- [X] I searched for [similar issues in this repository](https://github.com/mrousavy/react-native-vision-camera/issues) and found none.

**Comment by Filocava99** [comment]
I already tried adding the following code to my Podfile:
```swift
installer.target_installation_results.pod_target_installation_results.each do |pod_name, target_installation_result|
      target_installation_result.native_target.build_configurations.each do |config|
        # For third party modules who have React-bridging dependency to search correct headers
        config.build_settings['HEADER_SEARCH_PATHS'] ||= '$(inherited) '
        config.build_settings['HEADER_SEARCH_PATHS'] << '"$(PODS_ROOT)/Headers/Private/React-bridging/react/bridging" '
        config.build_settings['HEADER_SEARCH_PATHS'] << '"$(PODS_CONFIGURATION_BUILD_DIR)/React-bridging/react_bridging.framework/Headers" '
      end
    end
```
but it didn't work

**Comment by iSamuIsHere** [comment]
Same issue on React Native 0.72.3

**Comment by lucarinaldii** [comment]
Same issue for me

**Comment by SmokeLabapp** [comment]
same issue for me

**Comment by SmokeLabapp** [comment]
@Filocava99 were you able to fix the issue?

**Comment by Fungusjuice** [comment]
Same issue

Using Expo's EAS build
enabled "use_frameworks static" for react-native-firebase

"expo": "^49.0.0",
"react-native": "0.72.6",
"react-native-reanimated": "~3.6.1",
"react-native-vision-camera": "3.6.17",
"react-native-worklets-core": "^0.2.4",

"@react-native-firebase/analytics": "^18.6.0",
"@react-native-firebase/app": "^18.1.0"

**Comment by IlanAMG** [comment]
Same here...

**Comment by namti** [comment]
Same error here

**Comment by whentao** [comment]
same issue here

**Comment by Amurmurmur** [comment]
same issue

**Comment by Amurmurmur** [comment]
Solved here: https://github.com/mrousavy/react-native-vision-camera/issues/2255#issuecomment-1889014507

**Comment by sekhuat** [comment]
I also faced the same issue and I am using react-native-vision-camera ver 2.16.1. I solved the issue by adding `s.dependency "ReactCommon"` into the `node_modules/react-native-vision-camera/VisionCamera.podspec` at the last line as shown below.

```
Pod::Spec.new do |s|
  ...
  s.dependency "React-callinvoker"
  s.dependency "React"
  s.dependency "React-Core"
  s.dependency "ReactCommon"    <== Add this line
end
```

After made the changes, I execute `pod install` command then recompile the app again. Hope this will help those who using V2 of react-native-vision-camera.
